lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -240,29 +185,34 @@ export default function LaunchErrorDialog({
               account to one of this assignment&apos;s groups.
             </b>
           </p>
-        </BaseDialog>
+        </ErrorModal>
       );
 
     case 'error-fetching':
       // Do not display canned text if there is a back-end-provided message
       // to show here, as it's redundant and not useful
       return (
-        <BaseDialog busy={busy} error={error} onRetry={onRetry}>
+        <ErrorModal
+          busy={busy}
+          error={error}
+          onRetry={onRetry}
+          title="Something went wrong"
```

Comment:
The new `ErrorModal` requires a `title`, which I think is a good thing, as it helps surface here that this error state uses the default "Something went wrong" title and perhaps should not (something for follow-up).

Reply:
Yeah, ignore my comment, I don't even know what I meant.